trading/exchanges/rest_exchanges/rest_exchange.py

```diff
@@ -164,7 +169,12 @@ def get_client(self):
     # ORDERS
     async def get_order(self, order_id, symbol=None):
         if self.client.has['fetchOrder']:
-            self.get_personal_data().upsert_order(order_id, await self.client.fetch_order(order_id, symbol))
+            try:
+                updated_order = await self.client.fetch_order(order_id, symbol)
+                self.get_personal_data().upsert_order(order_id, updated_order)
+            except OrderNotFound:
+                # some exchanges are throwing this error when an order is cancelled (ex: coinbase pro)
```

Comment:
Its a normal behaviour for some exchanges ?

Reply:
apparently for coinbasepro it is